src/containers/other/NFTView/elements/Stats/Stats.tsx

```diff
@@ -99,7 +99,7 @@ export const Stats: React.FC<StatsProps> = ({
 							ethers.utils.formatEther(domainMetrics!.lastSale),
 					  ).toLocaleString() +
 					  ' ' +
-					  paymentTokenInfo.name
+					  paymentTokenInfo.symbol
```

Comment:
<img width="899" alt="Screenshot 2022-07-14 at 22 32 40" src="https://user-images.githubusercontent.com/39112648/179090642-96534ff4-cba5-4cda-a007-bd331dddc6b1.png">

Captured this while testing on Mainnet, but after a couple of refreshes and clicking around it displayed as expected

Reply:
@padala no, just tried on this branch and couldn't reproduce